### DISCUSSION TITLE
1

### DISCUSSION THREAD

**Discussion by tianmx6** [SwaggyMacro/YangLeGeYang#29] [Q&A] [answered]
羊助手怎末获得，可以详细说说吗

**Comment by SwaggyMacro** [comment, marked as answer]
[https://github.com/SwaggyMacro/YangLeGeYang/releases/download/0.5/SheepSheep.exe](https://github.com/SwaggyMacro/YangLeGeYang/releases/download/0.5/SheepSheep.exe)
现在只能刷话题通关次数了
